spectre-meltdown-checker.sh

```diff
@@ -1273,13 +1273,13 @@ read_cpuid()
 	# Linux returns it as decimal, BSD as hex, normalize to decimal
 	_reg=$(( _reg ))
 	# shellcheck disable=SC2046
-	_debug "cpuid: wanted register ($_register) has value $_reg aka "$(printf "%08x" "$_reg")
+	_debug "cpuid: wanted register ($_register) has value $_reg aka "$(printf "%08X" "$_reg")
 	_reg_shifted=$(( _reg >> _shift ))
 	# shellcheck disable=SC2046
-	_debug "cpuid: shifted value by $_shift is $_reg_shifted aka "$(printf "%x" "$_reg_shifted")
+	_debug "cpuid: shifted value by $_shift is $_reg_shifted aka "$(printf "%X" "$_reg_shifted")
 	read_cpuid_value=$(( _reg_shifted & _mask ))
 	# shellcheck disable=SC2046
-	_debug "cpuid: after AND $_mask, final value is $read_cpuid_value aka "$(printf "%x" "$read_cpuid_value")
+	_debug "cpuid: after AND $_mask, final value is $read_cpuid_value aka "$(printf "%X" "$read_cpuid_value")
```

Comment:
any reason for uppercasing all hexa?

Reply:
Hi, yes I was matching the cpuid output in FreeBSD which uses all uppercase for hexadecimal. However this is a minor change of course so can be excluded/removed/reverted.